Readme.md

```diff
@@ -68,13 +68,18 @@ const jsTargets = require('path/to/Lombiq.Gulp.Extensions/Tasks/js-targets');
 
 const path = './Assets/Scripts/'
 
-gulp.task('build:js', () => jsTargets.compile(path, './wwwroot/directory-to-copy-into'));
+const destination = './directory-to-cop-into'
+
+gulp.task('build:js', () => jsTargets.compile(path, destination));
 ```
+The build output or task runner will show you a list of all the ESLint rule violations.
```

Comment:
Meg a VS Error List window, meg `dotnet build`. Nézd át a .NET Analyzers guide-jait, az ottanihoz hasonlóan, még ha nem is pont ugyanúgy, megfelelő részletességgel, kellene itt is leírni, hogy hogyan kell ezeket használni: https://github.com/Lombiq/.NET-Analyzers#guides.